components/Smiley.tsx

```diff
@@ -19,12 +19,31 @@ const EMOJI_MAP: EmojiMap = {
 }
 
 const Smiley = ({ percentage }: Props) => {
+  const min = 0
+  const max = 10
   const matchingEmoji = Object.keys(EMOJI_MAP).find(
     (key) => parseInt(key) >= percentage * 100
   )
   if (!matchingEmoji) return null
 
-  return <span className={styles.large}>{EMOJI_MAP[matchingEmoji]}</span>
+  return (
+    <span
+      aria-label={`Satisfaction rating: ${
+        parseInt(matchingEmoji) / 10
+      }, ${min} is the worst rating, ${max} is the best rating`}
```

Comment:
Same issue as above with localization. I don't think a label is needed here either